sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -247,6 +247,12 @@ private[spark] object SQLConf {
           "otherwise the schema is picked from the summary file or a random data file " +
           "if no summary file is available.")
 
+  val PARQUET_SCHEMA_SKIP_MERGE_PARTFILES = booleanConf("spark.sql.parquet.skipMergePartFiles",
```

Comment:
I'd propose renaming this configuration to `spark.sql.parquet.respectSummaryFiles`.